tests/unit/test_sde.py

```diff
@@ -213,14 +229,47 @@ def test_ssm_to_linear_drift(setup):
     A_p = -1 * ou_sde.decay * tf.ones_like(m)[..., None] * dt + tf.eye(1, dtype=DTYPE)
     b_p = tf.zeros_like(m) * dt
 
-    p_ssm = state_space_model_from_covariances(initial_mean=x0, initial_covariance=1e-1 * tf.ones_like(x0[..., None]),
-                                               state_transitions=A_p,
-                                               state_offsets=b_p,
-                                               process_covariances=tf.square(ou_sde.diffusion(m, t=None)) * dt)
+    p_ssm = state_space_model_from_covariances(
+        initial_mean=x0,
+        initial_covariance=1e-1 * tf.ones_like(x0[..., None]),
+        state_transitions=A_p,
+        state_offsets=b_p,
+        process_covariances=tf.square(ou_sde.diffusion(m, t=None)) * dt,
+    )
     linear_drift = LinearDrift()
     linear_drift.set_from_ssm(p_ssm, dt=dt)
 
     true_A = -1 * ou_sde.decay * tf.ones_like(m)[..., None]
```

Comment:
this is a bit messy to read
true_A is the continuous drift while A_p is the discretized transitition of a SSM
maybe it would be good to reorganize the lines here to differentiate the two

like with docstrings
```
# continuous transition statistics
# discrete transition statistics
```